### ISSUE TITLE
Contradictions in bicycle attribute

### ISSUE THREAD

**Issue by following5** [opencaching/opencaching-pl#1244]
@harrieklomp @andrixnet 
At OC.nl and OC.ro there are contradicting definitions of the "bicycle" attribute.

OC.nl:
* English and German: "Bicycle recommended"
* Dutch and Polish "Can be reached by bicycle"

OC.ro:
* English "Bike"
* Romanian: "Bicycle recommended"
* Dush and Polish: "Can be reached by bicycle"

I think you should decide for one of both meanings. :-)

At OC.pl it is:
* English "Bike"
* Polish: "Can be reached by bicycle"

**Comment by harrieklomp** [comment]
Perhaps there could be made a standard English translation for all available attributes. This should include all attributes from all OC nodes.
This could be placed on https://opencaching.eu/index.php/Cache_attributes or somewhere else. Each node can add their own translation.
These translation can be used for databases and OKAPI

This is only a suggestion ;-)

**Comment by following5** [comment]
We have standard translations for all attributes [here](https://github.com/opencaching/okapi/blob/master/okapi/services/attrs/attribute-definitions.xml) in Okapi. But those may be wrong for the NL and RO "bicycle" attribute.

You must to decide what the meaning of this attribute is on your OC site. If it is "bicycle recommended", then a new Okapi attribute must be defined for that, which will have other meaning and translations than the PL attribute.

**Comment by harrieklomp** [comment]
If English translation and meaning in OKAPI is clear than all other translation should be the same.
I can only suggest (personal opinion) to make a standard English text and every node should translate it like it is. We can make a useless discussions about the bike attribute or any other attribute but we must make a standard.
If all English text in [OKAPI ](https://github.com/opencaching/okapi/blob/master/okapi/services/attrs/attribute-definitions.xml)are correct i will translate these in Dutch. I will use the same translations in database.

**Comment by harrieklomp** [comment]
Does any other node OCPL, OCUK, OCRO and (perhaps) OCUS agree with this?

**Comment by following5** [comment]
> If English translation and meaning in OKAPI is clear 

The meaning in Okapi is not clear for the NL and RO "bicycle" attributes. The may have been wrongly added to A27.

Please decide what is the _meaning_ of this attribute at OC.nl. Does it mean "accessible by bycycle" (likt at OC.pl), or "bicycle recommended"? These are different things.

**Comment by following5** [comment]
The bicycle attribute is the only one with this issue; all other are fine.

Look at http://www.opencaching.nl/okapi/devel/attrlist:

85: BIKE RECOMMENDED - A27
        de: Fahrrad benötigt
        en: Bike recommended
        nl: Bereikbaar per fiets
        pl: Dostępna rowerem

The "BIKE RECOMMENDED" is just a copy of the English entry, it has no special meaning. Which one gives the correct meaning of your attribute 85: The DE+EN text or the NL+PL text?

**Comment by andrixnet** [comment]
Any place accessible by bicycle is accessible by foot, right? Although covering the distance may require more time by foot.
Not all places accessible by bicycicle are accessible by car. 

Thus IMO "recommending bicycle" makes little sense. 
Ok, there may be a bicycle lane nearby / on the way. But still. Read below.

Furthermore, there may be permission restrictions against bicycles. This is very well represented by Groundspeak attribute 32 "Bicycles allowed", mapped by OKAPI in acode A27 and PL/RO/NL attribute 85.
I propose keeping this attribute as "Bicycles allowed" and modifying the OC translations accordingly.

**Comment by following5** [comment]
There is the same problem with the "Boat" attribute, which is "Boat recommended" at some OC sites and "Boat required" at other.

**Comment by following5** [comment]
Current naming of bike attribute in *local languages*:

* OC PL: Dostępna rowerem = accessible by bike - 19643 active caches
* OC NL: Bereikbaar per fiets = accessible by bike - 34 active caches
* OC RO: Recomandat cu bicicleta = bike recommended - 19 active caches
* OC UK: (has no bike attribute)

OKAPI definitions are equivalent to "accessible by bike" for all these languages. So I think we should go for that.

Current naming  of boat attribute in local languages:

* OC PL: Dostępna tylko łodzią = accessible only by boat - 153 active caches
* OC NL: Boot aanbevolen = boat recommended - 3 active caches
* OC RO: Ai nevoie de o barcă = boat required - 1 active cache
* OC UK: Boat required - 7 active caches

OKAPI definitions are equivalent to "(usually) accessible only by boat". So I think we should go for "boat required" here.

**Comment by following5** [comment]
In GC it is "Bicycles" and "Boat required". Looks like they also hat problems in defining the bike attribute.

**Comment by following5** [comment]
@harrieklomp 
> I can only suggest (personal opinion) to make a standard English text and every node should translate it like it is. 

Yeah, that was a good suggestion. These standard English texts are now in PR  #1852.

**Comment by andrixnet** [comment]
On the semantics of these attributes I think the following: 

- bike / bycicle = accessible by bycicle (you can ride a bycicle there) and bycicles allowed (permission)
IMO it can be used to show one or both meanings: 
* that is reachable by bycicle
* that bycicles are allowed. 

It is both an accesibility and permission attribute. 

example 1: path through the forest, you can ride your bike on it vs path through forest that is steep, is rocky, has fallen trees, it is impractical to ride a bycicle there.
example 2: city, a park where bycicles are allowed (some paths or entire park) vs park where bycicles are forbidden.

I would add:
- bycicles not allowed = it is forbidden to ride a bycicle there
(permission attribute only)

- boat = boat required. Accessible [only] by boat. 
This is an accessibility attribute. You can reach that place by boat, this being the easiest way. 
Swimming may be an option but most likely impractical. 
Flying may be an option but unlikely due to financial or legal reasons.
Not worth discussing other exotic alternatives, so yes, boat is required.
